pkgs/development/node-packages/default.nix

```diff
@@ -273,7 +273,7 @@ let
         wrapProgram "$out/bin/prisma" \
           --prefix PRISMA_MIGRATION_ENGINE_BINARY : "${prisma-engines}/bin/migration-engine" \
           --prefix PRISMA_QUERY_ENGINE_BINARY : "${prisma-engines}/bin/query-engine" \
-          --prefix PRISMA_QUERY_ENGINE_LIBRARY : "${lib.getLib prisma-engines}/libquery_engine.so.node"
+          --prefix PRISMA_QUERY_ENGINE_LIBRARY : "${lib.getLib prisma-engines}/libquery_engine.so.node" \
```

Comment:
```suggestion
          --prefix PRISMA_QUERY_ENGINE_LIBRARY : "${prisma-engines}/lib/libquery_engine.so.node" \
```
or add a lib output

Reply:
I was advised to use `lib.getLib` in the previous review...

Reply:
I think then this needs to create a lib output which it doesn't currently as far as I know. But that would probably be better. The file was not at that location as far as I can tell.

Reply:
getLib gracefully handles the case where there is no lib output so current situation is ok imo

Reply:
The problem is not that https://github.com/NixOS/nixpkgs/blob/a6fbb0b67ddf15f495c3fff0a05beb7885759863/lib/attrsets.nix#L495 doesn't fall back to $out but that the **path** is wrong. For me it is in the `lib` folder and not at the root.

Reply:
ah my bad I had not seen that

Reply:
@Pamplemousse If the path is also different for you just the path change would be enough in my opinion.